app/src/main/res/layout/popular_movie_recycler_item_layout.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Is it needed? `CardView` probably could be a top-parent, try to create as few nestings as possible.